### ISSUE TITLE
Lost root on reboot

### ISSUE THREAD

**Issue by XploitWizer** [arpruss/cve2019-2215-3.18#4]
Apk successfully rooted Redmi Note 4 mido with locked bootloader but i get lost root and need to reinstall after reboot and also please help in turning off secure boot on locked bootloader using this root

Thanks for such a constant and great work

**Comment by arpruss** [comment]
Can you run install again after the reboot?

**Comment by XploitWizer** [comment]
still lost root after reboot please also work on disabling boot verification and unlocking bootloader too
thanks

**Comment by arpruss** [comment]
Do you have the "install on boot" checkmark checked?
And what does su98 say after a reboot about /sbin/su98?

**Comment by XploitWizer** [comment]
yes checkmark done but after reboot i lost root accces when i check in root checker app and when i open your apk it will ask me to install again shows that not installed
